libs/sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -564,6 +565,25 @@ public Node visitDropUser(SqlBaseParser.DropUserContext context) {
         );
     }
 
+    @Override
+    public Node visitCreateRole(SqlBaseParser.CreateRoleContext ctx) {
+        List<String> options = new ArrayList<>();
+        if (ctx.spaceSeparatedIdents() != null) {
+            options = identsToStrings(ctx.spaceSeparatedIdents().ident());
+            for (String option: options) {
+                if (CreateRole.INHERIT.equals(option) == false) {
+                    throw new IllegalArgumentException(
+                        String.format(Locale.ENGLISH, "CREATE ROLE statement doesn't support option '%s'", option)
+                    );
+                }
+            }
+        }
+        return new CreateRole<>(
+            getIdentText(ctx.name),
+            options
+        );
+    }
```

Comment:
We can actually drop options list from `CreateRole` since in the first iteration we anyway don't support any option other than inherit and it will be default.

For now I kept it in `CreateRole` to be future proof - in case we support NOINHERIT, LOGIN and others, mentioned in https://www.postgresql.org/docs/14/sql-createrole.html

Reply:
It's fine to keep, but maybe it's better to validate this in analyzer?

Reply:
Agree, let's keep parser simple. Removed validation